arangod/Transaction/Methods.cpp

```diff
@@ -1543,7 +1543,11 @@ Future<OperationResult> transaction::Methods::insertLocal(std::string const& cna
     // Block operation early if we are not supposed to perform it:
     auto const& followerInfo = collection->followers();
     std::string theLeader = followerInfo->getLeader();
-    if (theLeader.empty()) {
+    bool leaderTouched = followerInfo->getLeaderTouched();  // this is false after a restart
+    if (collection->name() == "_statisticsRaw" || collection->name() == "_statistics15" || collection->name() == "_statistics") {
```

Comment:
can we change this to `collection->name().compare(0, 11, "_statistics") == 0` or something alike?